components/TestSubmissionsResults/TestSubmissionsResults.module.scss

```diff
@@ -0,0 +1,27 @@
+﻿@import '../../pageStyles/constants';
+
+.listItem{
+  font-family: $softwireFont;
+  font-size: $contentFontSize;
+  color: $softwireWhite;
+  @include sharedListStyle;
```

Comment:
I think the `list-style` applies to the list itself, not the list item. I don't think that has any impact here.